test/parallel/test-stream-readable-flow-no-flow-mixed.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+const common = require('../common');
+
+// This test ensures that Readable stream switches between flowing and
+// non-flowing state properly when varying the 'readable' and 'data' event
+// subscription.
```

Comment:
I just would like to avoid regressions with `hmw: 0`, it should be mentioned here.